build-config/make/u-boot.make

```diff
@@ -139,7 +139,7 @@ $(UBOOT_BUILD_STAMP): $(UBOOT_IMAGE)
 u-boot-install: $(UBOOT_INSTALL_STAMP)
 $(UBOOT_INSTALL_STAMP): $(UBOOT_BUILD_STAMP)
 	$(Q) echo "==== Installing u-boot ($(MACHINE_PREFIX)) ===="
-	$(Q) cp -v $(UBOOT_IMAGE) $(UBOOT_INSTALL_IMAGE)
+	$(Q) cp -v $(UBOOT_BUILD_DIR)/$(UBOOT_MACHINE)/u-boot-dtb.bin $(UBOOT_INSTALL_IMAGE)
```

Comment:
In this same file, `u-boot.make`, look up towards the beginning for `UBOOT_PBL_ENABLE`.  When that is `yes` the name of the u-boot binary name is different.  We can do something similar here.

How about if `UBOOT_DTB_ENABLE` is yes then the binary is `u-boot-dtb.bin`.  If it is not then use the legacy name `u-boot.bin`.

A particular machine can set `UBOOT_DTB_ENABLE` in its `machine.make` file.